invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/footer.html

```diff
@@ -17,40 +17,40 @@
         <div class="rdm-footer five wide tablet two wide computer column">
           <h2 class="ui small header">{{ _("About InvenioRDM") }}</h2>
           {% trans invenio_rdm="http://inveniosoftware.org/products/rdm" %}
```

Comment:
You're correct, using `{% trans %}` here doesn't make much sense since URLs don't change based on the language, not sure why it's been added in the first place. I'll remove those blocks to clean up the code. Appreciate the feedback, please find more! :)